packages/db/postgres/migrations/009-migrate-backup-urls.sql

```diff
@@ -0,0 +1 @@
+UPDATE "upload" SET "backup_urls" = (SELECT array_agg(jsonb_build_object('url', url, 'inserted_at', inserted_at)) FROM backup WHERE "upload_id" = "upload"."id");
```

Comment:
Nit.
Add new line at the end of the file.

I've noticed this in other files, possibly something you want to set up in your editor.